http/tus_handlers.go

```diff
@@ -147,8 +147,7 @@ func tusPostHandler() handleFunc {
 		// Enables the user to utilize the PATCH endpoint for uploading file data
 		registerUpload(file.RealPath(), uploadLength)
 
-		// Signal the frontend to reuse the current request URL
-		w.Header().Set("Location", "")
+		w.Header().Set("Location", "/api/tus"+r.URL.Path)
```

Comment:
```suggestion
		w.Header().Set("Location", d.server.BaseURL+"/api/tus"+r.URL.Path)
```

Reply:
Is there a need to add the server URL? Does the JS library not resolve the absolute URL in relation to the root URL?

Reply:
The frontend will resolve the URL by joining origin (http://localhost:8080) with the location value (/baseurl/api/tus/file.test).